src/Rule/DateTime.php

```diff
@@ -1,160 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Yiisoft\Validator\Rule;
-
-use Closure;
-use Attribute;
-use InvalidArgumentException;
-use Yiisoft\Validator\WhenInterface;
-use Yiisoft\Validator\DumpedRuleInterface;
-use Yiisoft\Validator\SkipOnErrorInterface;
-use Yiisoft\Validator\SkipOnEmptyInterface;
-use Yiisoft\Validator\Rule\Trait\WhenTrait;
-use Yiisoft\Validator\RuleHandlerInterface;
-use Yiisoft\Validator\Rule\Trait\SkipOnEmptyTrait;
-use Yiisoft\Validator\Rule\Trait\SkipOnErrorTrait;
-
-/**
- * Defines validation options to check that the value is a date.
- *
- * An example for simple  that can be used to validate the date:
- * ```php
- * use Yiisoft\Validator\Rule\DateTime;
- *
- * $rules = [
- *      'date' => [
- *          new DateTime(format: 'Y-m-d'),
- *      ],
- * ];
- * ```
- * In the example above, the PHP attributes equivalent will be:
- *
- * ```php
- * use Yiisoft\Validator\Validator;
- * use Yiisoft\Validator\Rule\DateTime;
- *
- * final class User
- * {
- *     public function __construct(
- *          #[DateTime(format: 'Y-m-d')]
- *          public string $date,
- *     ){
- *    }
- * }
- *
- * $user = new User( date: '2022-01-01' );
- *
- * $validator = (new Validator())->validate($user);
- *
- * ```
- *
- * @see DateTimeHandler
- *
- * @psalm-import-type SkipOnEmptyValue from SkipOnEmptyInterface
- * @psalm-import-type WhenType from WhenInterface
- */
-#[Attribute(Attribute::TARGET_PROPERTY | Attribute::IS_REPEATABLE)]
-final class DateTime implements DumpedRuleInterface, SkipOnErrorInterface, WhenInterface, SkipOnEmptyInterface
```

Comment:
Removing the rule?

Reply:
It was moved to `Date` folder, this is a copy.